docs/components/detail/abbreviations.html

```diff
@@ -3232,10 +3232,21 @@ <h4 class="Tree-collectionLabel" data-role="toggle">
         
     
         
+        <li class="Tree-item Tree-collection Tree-depth-3" data-behaviour="collection" id="tree-components-collection-side-nav">
+            <h4 class="Tree-collectionLabel" data-role="toggle">
+                <span>Side Nav</span>
+            </h4>
+            <ul class="Tree-collectionItems" data-role="items">
+            
```

Comment:
@adrianconte there is a lot of vertical spacing in here. Could you please check the formatting and update accordingly in each relevant file.